PurchasesTests/TestHelpers/RCObjC.h

```diff
@@ -0,0 +1,19 @@
+//
+//  RCObjC.h
+//  Purchases
+//
+//  Created by Joshua Liebowitz on 6/10/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RCObjC : NSObject
+
++ (BOOL)catchException:(void(^)(void))tryBlock error:(__autoreleasing NSError **)error;
```

Comment:
should this method name reference its first param? we can always use `NS_SWIFT_NAME` to skip its name in swift

Reply:
I'm not strongly opposed to either way, I'll update and see what you think about it!